core/src/test/java/com/linecorp/armeria/common/logging/DefaultRequestLogTest.java

```diff
@@ -274,4 +275,18 @@ public void toStringResponseBuilderCapacity() {
                 responseTrailers.toString().length() +
                 cause.toString().length());
     }
+
+    @Test
+    public void testUUID_Assign() {
+        final RequestHeaders reqHeaders =
+                RequestHeaders.of(HttpMethod.POST, "/armeria/uuid",
+                                  HttpHeaderNames.CONTENT_LENGTH, VERY_LONG_STRING.length());
+        final HttpRequest req = HttpRequest.of(
+                AggregatedHttpRequest.of(reqHeaders, HttpData.ofUtf8(VERY_LONG_STRING)));
+        final ClientRequestContext cctx = ClientRequestContextBuilder.of(req).build();
+        assertThat(cctx.log().uuid()).isNotNull();
+
+        final ServiceRequestContext sctx = ServiceRequestContext.of(req);
+        assertThat(sctx.log().uuid()).isNotNull();
```

Comment:
Could ensure `ctx.log().uuid()` is equal to `ctx.uuid()` as well as checking non-nullness.